ees/eereq.go

```diff
@@ -33,17 +33,25 @@ import (
 
 // NewEventExporterRequest returns a new EventExporterRequest
 func NewEventExporterRequest(req utils.DataProvider, dc utils.MapStorage,
-	tnt, timezone string, filterS *engine.FilterS) (eeR *EventExporterRequest) {
+	tntTpl config.RSRParsers,
+	dfltTenant, timezone string, filterS *engine.FilterS) (eeR *EventExporterRequest) {
 	eeR = &EventExporterRequest{
 		req:     req,
 		tmz:     timezone,
-		tnt:     tnt,
 		filterS: filterS,
 		cnt:     utils.NewOrderedNavigableMap(),
 		hdr:     utils.NewOrderedNavigableMap(),
 		trl:     utils.NewOrderedNavigableMap(),
 		dc:      dc,
 	}
+	// populate tenant
```

Comment:
For simplicity, please define default tenant directly within eeR and overwrite if present in tntTpl